src/ImageIO/CGImageAnimation.cs

```diff
@@ -0,0 +1,143 @@
+//
+// ImageIO - CGImageAnimation.cs
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020, Microsoft Corp.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace ImageIO
+{
+
+    public partial class CGImageAnimation
```

Comment:
This is a new type right ?

Since there's nothing for the instance then it makes no sense to have a way to instantiate it.

Make this a static type and all it's members static.